dotCMS/src/main/java/com/dotcms/rest/config/RestServiceUtil.java

```diff
@@ -1,37 +1,14 @@
 package com.dotcms.rest.config;
 
-import com.dotcms.rest.servlet.ReloadableServletContainer;
-
-
-
 public class RestServiceUtil {
-	
-	public synchronized static void addResource(Class clazz){
-		
-		new DotRestApplication().getClasses();
-		if(DotRestApplication.REST_CLASSES.contains(clazz)){
-			return;
-		}
-		
-		DotRestApplication.REST_CLASSES.add(clazz);
 
-		reloadRest();
-		
+    public static void addResource(Class clazz) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'clazz' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5566474)